packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.types.ts

```diff
@@ -196,6 +196,10 @@ export interface IDetailsHeaderStyles {
   check: IStyle;
   cellWrapperPadded: IStyle;
   cellIsCheck: IStyle;
+
+  /**
+   * @deprecated Not used
+   */
   cellIsActionable: IStyle;
   cellIsEmpty: IStyle;

```

Comment:
Similarly, is `cellIsEmpty` unused? Also, `cellWrapperPadded`? If so, perhaps we should include those as part of this deprecations pass.

Reply:
Good catch! I can deprecate those here too

Reply:
Might as well deprecate all that are unused. I was honestly surprised to stumble upon that many being unused, hope I'm not missing something..